controllers/binding/snapshotenvironmentbinding_controller.go

```diff
@@ -114,7 +114,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	adapter := NewAdapter(snapshotEnvironmentBinding, snapshot, environment, application, component, integrationTestScenario, logger, loader, r.Client, ctx)
 
 	return reconciler.ReconcileHandler([]reconciler.ReconcileOperation{
-		adapter.EnsureIntegrationTestPipelineForScenarioExists,
+		adapter.EnsureIntegrationTestPipelineForScenarioExists, adapter.EnsureEphemeralEnvironmentsCleanedUp,
```

Comment:
If you don't mind, can we please add this latest adapter method in a new line, so it stays consistent with how other adapters are defined within their controller?